File: u01/afg2.py
import plotly.express as px
import pandas as pd

data = {
    "Names": ["Informatik", "überfachlichte Kompetenz", "Physik, Medizin, Biologie", "Medizininformatik", "Mathematik"],
    "Values": [28, 11.5, 25, 25, 10.5]
}

fig1 = px.pie(data, names='Names', values='Values')

fig1.show()

File: u01/afg2.js
var data = [
    {
        values: [28, 11.5, 25, 25, 10.5],
        labels: [
            "Informatik",
            "überfachliche Kompetenz",
            "Physik, Medizin, Biologie",
            "Medizininformatik",
            "Mathematik",
        ],
        type: "pie",
    },
];

var layout = {
    height: 400,
    width: 500,
};

Plotly.newPlot("myDiv", data, layout);
